src/index.js

```diff
@@ -477,6 +478,16 @@ export default class Carousel extends React.Component {
       case 40:
         this.previousSlide();
         break;
+      case 32:
+        if (this.state.pauseOnHover && this.props.autoplay) {
```

Comment:
@sarmeyer I tested this out but since you know a little more of the use cases for the carousel let me know if it actually works.